feedback.sh

```diff
@@ -1,5 +1,8 @@
 #! /bin/sh
 
+echo feedback.sh is useless while we transition to blackboard, and will probably get deleted
```

Comment:
to brightspace

Reply:
Good idea, I've made blackboard-final a tag.